akka-cluster/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>INFO</level>
+        </filter>
+        <encoder>
+            <pattern>%date{ISO8601} %-5level %logger %marker %X{akkaSource} %X{sourceThread} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="stash" class="net.logstash.logback.appender.LogstashTcpSocketAppender">
+        <destination>127.0.0.1:12345</destination>
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
+    </appender>
```

Comment:
This is just temporary testing of integrating with ELK stack, will be removed before merging.

I used standard installation of Elasticsearch and Kibana. Logstash started with tcp input `bin/logstash -f config/logstash-tcp.conf`:

```

input {
  tcp {
    port => 12345
    codec => json
  }
}

output {
  elasticsearch {
    hosts => ["http://localhost:9200"]
    #user => "elastic"
    #password => "changeme"
  }
}
```

The marker name shows up as `tags` in Kibana, and MDC properties are also there:

<img width="2073" alt="Screenshot 2019-11-20 at 12 42 34" src="https://user-images.githubusercontent.com/336161/69238211-bc93ee00-0b97-11ea-8fb8-22c7973fb40f.png">